packages/connectivity/lib/connectivity_tizen.dart

```diff
@@ -0,0 +1,18 @@
+// You have generated a new plugin project without
```

Comment:
I guess this file was added accidentally.

Reply:
Thanks.